### PR TITLE
bump version to 0.27

### PR DESCRIPTION
**Pull request by gdementen**

### CODE REVIEW

**Comment by alixdamman** [comment]
I need to test conda build for Python 3+ on my Linux. There is something wrong but what... ?

**Comment by gdementen** [comment]
@alixdamman: Unsure it is caused by conda build. It might be related to condaforge config for circleci/how it calls tests. Somehow it fails.

**Comment by gdementen** [comment]
oh, this might simply be the pandas version problem!

**Comment by gdementen** [comment]
maybe this needs to be re-rendered using a more recent version of conda-smithy as in
https://github.com/conda-forge/larray-editor-feedstock/pull/2???

File: README.md [hunk 1]
```diff
@@ -14,9 +14,9 @@ Summary: Graphical User Interface for LArray.
 Current build status
 ====================
 
-Linux: [![Circle CI](https://circleci.com/gh/conda-forge/larray-editor-feedstock.svg?style=shield)](https://circleci.com/gh/conda-forge/larray-editor-feedstock)
-OSX: [![TravisCI](https://travis-ci.org/conda-forge/larray-editor-feedstock.svg?branch=master)](https://travis-ci.org/conda-forge/larray-editor-feedstock)
-Windows: [![AppVeyor](https://ci.appveyor.com/api/projects/status/github/conda-forge/larray-editor-feedstock?svg=True)](https://ci.appveyor.com/project/conda-forge/larray-editor-feedstock/branch/master)
+Linux: [![Circle CI](https://circleci.com/gh/conda-forge/build-feedstock.svg?style=shield)](https://circleci.com/gh/conda-forge/build-feedstock)
```
Comment (gdementen) [inline README.md]: this seems wrong to me...

**Comment by CJ-Wright** [comment]
@conda-forge-admin, please rerender

**Pull request closed without merging**